### DISCUSSION TITLE
If I get a low TSS enrichment, how can I filter it effectively?

### DISCUSSION THREAD

**Discussion by Melody-cell** [aertslab/pycisTopic#189] [Q&A]
I got a low TSS enrichment on my data, i wonder how can I make an effective filtering, 
cause i don't see any filtering steps on this new version of this tutorial:
![image](https://github.com/user-attachments/assets/1664f787-7e64-4262-a64f-22c614e0ab00)
I wonder if I can filter cells lower than the thresholds, and using cells after filtered to plot a qc plot?
Did anyone know how to solve it?
Thank you for your consideration.

**Comment by SeppeDeWinter** [comment]
Hi @Melody-cell 

If you get a low TSS enrichment, you also have to make sure that you are using a correct genome annotation. The TSS calculation depends on getting TSS location from the genome annotation.

For filtering thresholds, please see the QC section in the [tutorial](https://pycistopic.readthedocs.io/en/latest/notebooks/human_cerebellum.html#QC). You can set the threshold in the `get_barcodes_passing_qc_for_sample ` function.

I hope this helps?

All the best,

Seppe
